dm/config/subtask.go

```diff
@@ -186,6 +186,8 @@ type SubTaskConfig struct {
 
 	ConfigFile string `toml:"-" json:"config-file"`
 
+	RemoveFinishedDump bool `toml:"remove-finished-dump" json:"remove-finished-dump"`
```

Comment:
This name looks not clear, is there a better name?

Reply:
clean-dump-file?

Reply:
I think it's OK